stability-tests/mempool-limits/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"github.com/kaspanet/kaspad/infrastructure/network/rpcclient"
+	"github.com/kaspanet/kaspad/stability-tests/common"
+	"github.com/kaspanet/kaspad/util/panics"
+	"github.com/kaspanet/kaspad/util/profiling"
+	"github.com/pkg/errors"
+	"os"
+)
+
+const mempoolSizeLimit = 1_000_000
+
+func main() {
+	defer panics.HandlePanic(log, "mempool-limits-main", nil)
+	err := parseConfig()
+	if err != nil {
+		panic(errors.Wrap(err, "error in parseConfig"))
+	}
+	defer backendLog.Close()
+	common.UseLogger(backendLog, log.Level())
+
+	cfg := activeConfig()
+	if cfg.Profile != "" {
+		profiling.Start(cfg.Profile, log)
+	}
+
+	defer func() {
+		if err := recover(); err != nil {
+			log.Criticalf("mempool-limits failed: %s", err)
```

Comment:
Instead of "failed" better write "paniced". Basically the same, but conveys info much more accurately.